### DISCUSSION TITLE
Is it possible to deploy 1.0.77 to the Maven repository?

### DISCUSSION THREAD

**Discussion by pwgit-create** [amithkoujalgi/ollama4j#59] [Q&A] [answered]
Congratulations on your excellent work on from version 1.0.73 to 1.0.77!  :snowflake: 👍 

It would be great if you could upload the latest version of Maven, which is 1.0.77, to the Mvn repository. Is it within the realm of possibility? Unlike the maven repository, there is no control over who owns the packages on jitpack, which makes it easier for malicious actors to install bad code on that platform.  In any case, your work is fantastic. It has played a significant role in the development of Ollama. Thank you, it goes without saying! :100:

https://github.com/gradle/gradle/issues/16310

**Comment by amithkoujalgi** [comment, marked as answer]
Thank you so much for your kind words! 🙏🏽 I'm thrilled to hear that the updates to the library have been well-received. Your feedback means a lot, and it motivates me to keep improving. 🏃🏼

I appreciate you pointing out the potential issues with package publishing!

Your suggestion has been immensely valuable, and I took it very seriously. I have made the necessary changes to push the artifacts to both Maven Central and GitHub Packages (which are the trusted repositories). Additionally, I have updated the README file to remove the Jitpack-based usage instructions.

From now on, [Maven Central](https://central.sonatype.com/artifact/io.github.ollama4j/ollama4j) would be our primary repo for publishing packages.

Version `1.0.78` is available to use from Maven Central.

```xml
<dependency>
    <groupId>io.github.ollama4j</groupId>
    <artifactId>ollama4j</artifactId>
    <version>1.0.78</version>
</dependency>
```

In the upcoming releases, I will be updating the package structure from `io.github.amithkoujalgi.*` to `io.github.ollama4j.*`.

**Reply by pwgit-create** [reply]
Thank you :smiley: I can confirm that version 1.0.79 has been uploaded to Maven!
